asaps/models.py

```diff
@@ -0,0 +1,132 @@
+from asnake.client import ASnakeClient
+import importlib
+import json
+import attr
+
+
+@attr.s
+class Client:
+    authclient = attr.ib
```

Comment:
Remove this authclient attribute from here, we'll set it in the `__init__` method.